Userland/cal.cpp

```diff
@@ -1,21 +1,24 @@
 #include <stdio.h>
 #include <time.h>
 
-int dayOfWeek(int day, int month, int year)
+int day_of_week(int day, int month, int year)
 {
-    const int seekTable[] = { 0, 3, 2, 5, 0, 3, 5, 1, 4, 6, 2, 4 };
+    static const int seek_table[] = { 0, 3, 2, 5, 0, 3, 5, 1, 4, 6, 2, 4 };
     if (month < 3)
         --year;
 
-    return (year + year / 4 - year / 100 + year / 400 + seekTable[month - 1] + day) % 7;
+    return (year + year / 4 - year / 100 + year / 400 + seek_table[month - 1] + day) % 7;
 }
 
-int getNumberOfDays(int month, int year)
+int get_number_of_days(int month, int year)
 {
+    bool is_leap_year = ((year % 400 == 0) || (year % 4 == 0 && year % 100 != 0));
+    bool is_long_month = (month == 1 || month == 3 || month == 5 || month == 7 || month == 8 || month == 10 || month == 12);
+
     if (month == 2)
-        return ((year % 400 == 0) || (year % 4 == 0 && year % 100 != 0)) ? 29 : 30;
+        return is_leap_year ? 29 : 30;
```

Comment:
Shouldn't this be `return is_leap_year ? 29 : 28;`?

Reply:
Oops! My brain did ++ instead of --. Fixed.